src/OrbitGl/AccessibleCaptureViewElement.cpp

```diff
@@ -22,36 +22,35 @@ orbit_accessibility::AccessibilityRect AccessibleCaptureViewElement::AccessibleR
   Vec2 pos = capture_view_element_->GetPos();
   Vec2 size = capture_view_element_->GetSize();
 
-  Vec2i screen_pos = viewport->WorldToScreenPos(pos);
-  int screen_width = viewport->WorldToScreenWidth(size[0]);
-  int screen_height = viewport->WorldToScreenHeight(size[1]);
+  Vec2i screen_pos = viewport->WorldToScreen(pos);
+  Vec2i screen_size = viewport->WorldToScreen(size);
```

Comment:
I assume this only works because actually the lower right corner is transformed and not a size? This tripped me a bit but I'm not sure if it warrants a rename or a commit. But at least I wanted to hear your opinion.